packages/docs/src/components/styleguide.tsx

```diff
@@ -6,77 +6,59 @@
  */
 
 import * as classNames from "classnames";
+import { IPageData, IPageNode, isPageNode } from "documentalist/dist/client";
```

Comment:
then maybe at least we should create a stub at the root instead so that we can import from `"documentalist/client"`